TinyMCE.Blazor/wwwroot/TinyMce.Blazor.lib.module.js

```diff
@@ -1,4 +1,4 @@
-console.log('loading js tinymce-blazor');
+﻿console.log('loading js tinymce-blazor');
```

Comment:
Just confirming that you intend this `console.log` to be here.

Reply:
Yeah. That's there for people to know if the library loaded in their project since they can forget to add it